src/collision/collider/mod.rs

```diff
@@ -86,6 +242,41 @@ pub trait AnyCollider: Component + ComputeMassProperties {
     ) -> Vec<ContactManifold>;
 }
 
+/// A simplified wrapper around [`AnyCollider`] that doesn't require passing in the context for
+/// implementations that don't need context
+pub trait SimpleCollider: AnyCollider<Context = ()> {
+    /// [`AnyCollider::aabb_with_context`] but without context
+    fn aabb(&self, p: Vector, r: impl Into<Rotation>) -> ColliderAabb {
```

Comment:
Given that these are the simpler versions of the methods, and what most people will probably reach for first unless specifically dealing with contexts, I think these should have proper docs like the methods on `AnyCollider`.

Could just copy them over and have some comment mentioning the context version, maybe something like this?

```suggestion
    /// Computes the [Axis-Aligned Bounding Box](ColliderAabb) of the collider
    /// with the given position and rotation.
    #[cfg_attr(
        feature = "2d",
        doc = "\n\nThe rotation is counterclockwise and in radians."
    )]
    ///
    /// For a version that provides access to an ECS context, see [`AnyCollider::aabb_with_context`].
    fn aabb(&self, p: Vector, r: impl Into<Rotation>) -> ColliderAabb {
```

Reply:
I'd also use full names for the arguments, there's not really a reason to abbreviate here.

These methods could also be inlined, though it's probably done automatically anyway.